src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -213,7 +241,56 @@ export default function ApplicationForm({
     type: "submit"
   };
 
-  const applicationVerification = () => {};
+  const applicationVerification = () => {
+    if (birthPlace === "") {
```

Comment:
can this just be `if birthPlace`

Reply:
Or you mean if !birthplace?

These are checking if nothing was input in the field. If they are empty, set an error.

Reply:
Fixed, and improved how input is handled to reset errors!